index.js

```diff
@@ -0,0 +1,46 @@
+import displayBook from './modules/displayBook.js';// eslint-disable-line
+import { setMyLib, getMyLib } from './modules/localStorage.js';// eslint-disable-line
+import navigate from './modules/navigation.js';// eslint-disable-line
+import displayDateTime from './modules/addDate.js';// eslint-disable-line
```

Comment:
- [ ] You are not allowed to disable linter, Linter is a tool for identifying and reporting errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows, with the goal of making code more consistent and avoiding bugs. Kindly fix this instead of disabling it 👍🏻